ooiui/static/js/views/common/MapView.js

```diff
@@ -164,25 +164,23 @@ var MapView = Backbone.View.extend({
         return gliderTrackLayer;
     },
     getRandomColor:function(){
-
+        var hue2rgb = function (p, q, t){
+            if(t < 0) t += 1;
+            if(t > 1) t -= 1;
+            if(t < 1/6) return p + (q - p) * 6 * t;
+            if(t < 1/2) return q;
+            if(t < 2/3) return p + (q - p) * (2/3 - t) * 6;
+            return p;
+        };
         var golden_ratio_conjugate = 0.618033988749895;
         var h = Math.random();
 
         var hslToRgb = function (h, s, l){
             var r, g, b;
 
-            if(s == 0){
+            if(s === 0){
```

Comment:
@FBRTMaka thanks, im familiar with the syntax, i was wondering why we changed it.